docs/data/charts/localization/localization.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Charts - Localization
+productId: x-charts
+components: ChartsLocalizationProvider, ChartsDataContainer, ChartsLocalizationProvider
+---
+
+# Charts - Localization
+
+<p class="description">The Charts allows to support users from different locales, with formatting, and localized strings.</p>
+
+The default locale of MUI X is English (United States). If you want to use other locales, follow the instructions below.
+
+## Translation keys
+
+You can use the `localeText` prop to pass in your own text and translations.
+You can find all the translation keys supported in [the source](https://github.com/mui/mui-x/blob/-/packages/x-charts/src/constants/defaultLocale.ts)
+in the GitHub repository.
+In the following example, the labels of the loading overlay are customized.
+
+{{"demo": "CustomLocaleOverlay.js", "bg": "inline"}}
+
+## Locale text
+
+You can use the theme to configure the locale text:
+
+```jsx
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { frFR } from '@mui/x-charts/locales';
+// Or import { frFR } from '@mui/x-charts-pro/locales';
+
+const theme = createTheme(
+  {
+    palette: {
+      primary: { main: '#1976d2' },
+    },
+  },
+  frFR,
+);
+
+<ThemeProvider theme={theme}>
+  <BarChart />
+</ThemeProvider>;
+```
+
+Note that `createTheme()` accepts any number of arguments.
+If you are already using the [translations of the core components](/material-ui/guides/localization/#locale-text), you can add `frFR` as a new argument.
+The same import works for Charts Pro as it's an extension of Charts.
+
+```jsx
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { frFR } from '@mui/x-charts/locales';
```

Comment:
Might be clearer

```suggestion
import { chartsfrFR } from '@mui/x-charts/locales';
```

Reply:
That would break the logic  we use on all packages and their related example.

For now, all packages export locales as follow:

```jsx
import { xxYY } from '@mui/[packageName]/locales';
```

And for multiple packages, we encourage them to do
```jsx
import { xxYY as [packageName]XxYY } from '@mui/[packageName]/locales';
```

Reply:
While doing the update, I understood your point. Exporting an object that contained only the `localeText` content.

I propose to export `frFR` and `frFRLocaleText` to be more explicit that `chartsfrFR` that does not says if the content is made for `localText` prop of the `createTheme()`